src/test/kotlin/com/igorwojda/string/decapitalizeconst/challenge.kt

```diff
@@ -4,7 +4,9 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun decapitalizeConst(str: String): String {
-    TODO("not implemented")
+    var newStr= str.replace("_"," ").trimStart().split(" ").map{ it.toLowerCase() }.map{ it.capitalize() } as ArrayList<String>
```

Comment:
nitpick: chains of functions calls should be formatted on multiple lines for better readibility:
```
str.replace("_"," ")
    .trimStart().split(" ")
    .map{ it.toLowerCase() }
    .map{ it.capitalize() }
```